completion/available/git.completion.bash

```diff
@@ -624,7 +624,13 @@ __git_commands () {
 	then
 		printf "%s" "${GIT_TESTING_COMMAND_COMPLETION}"
 	else
-		git help -a|egrep '^  [a-zA-Z0-9]'
+		git_cmd_list=$(git help -a|egrep '^  [a-zA-Z0-9]')
```

Comment:
If I'm not mistaken, this will run the `git help ... egrep` command in all cases, not only as a fallback. This means that in the case of Git 2.18+, the code is executed twice, once through `git help` and then again through `git --list-cmds`.